webapp/components/definition/connectivity/NetPyNEConnectivityRule.js

```diff
@@ -32,6 +32,10 @@ export default class NetPyNEConnectivityRule extends React.Component {
     };
   }
 
+  UNSAFE_componentWillReceiveProps (nextProps) {
+    this.setState({ currentName: nextProps.name });
```

Comment:
A comment here about what this fixes would be nice

Reply:
not a fix, simply the linter was complaining for the declaration order of the functions and I adjusted that since I was tired of the minefield, check below and you will see this function. Thx